### DISCUSSION TITLE
Auto/Rapid fire effect.

### DISCUSSION THREAD

**Discussion by darthvaderx** [AntiMicroX/antimicrox#1023] [Q&A]
I'm trying to add the auto/rapid fire effect with an XBox One controller on an emulator that doesn't have this feature, I've already tried several macro combinations in AntiMicroX without any success, what should I do to be able to perform this task, thank you.

**Comment by anomadeus** [comment]
The same here. If this software can't do this, anyone know one that can? Thanks

**Reply by darthvaderx** [reply]
I don't know what I did, but somehow I managed to auto-fire with this software in the Atomic Robo-Kid game, the problem is that I didn't save it and I don't know what configuration I used anymore.
